spyder/plugins/help/widgets.py

```diff
@@ -349,6 +353,18 @@ def setup(self, options):
             toggled=lambda value: self.set_option('wrap', value),
             initial=self.get_option('wrap'),
         )
+        self.copy_action = self.create_action(
+            name="help_plain_text_copy_action",
```

Comment:
Why is this action not listed in an enum class, like in other plugins moved to the new API?

Reply:
Fixed.